packages/api-tests/test/test-helpers.js

```diff
@@ -4,9 +4,37 @@ const executeAsyncJavascript = (client, script, ...args) => {
   return client.executeAsync(script, ...args);
 };
 
-const selectWindow = (client, handle) => {
+let addedHandles = [];
+const initialiseWindows = (client) => {
+  const script = (callback) => {
+    if (window.ssf) {
+      ssf.app.ready().then(() => {
+        callback();
+      });
+    } else {
+      callback();
+    }
+  };
+
+  // Get a list of any extra window handles that get create during initialisation
+  return executeAsyncJavascript(client, script)
+      .then(() => client.windowHandles())
+      .then(handles => {
+        addedHandles = handles.value.splice(1);

```

Comment:
Probably worth adding a comment regarding why the first window handle is removed.